### PR TITLE
Media Support for Wikis

### PR DESCRIPTION
**Pull request by Royal-lobster**
## [DRAFT]
Fixes https://github.com/EveripediaNetwork/issues/issues/327
Closes https://github.com/EveripediaNetwork/issues/issues/305

# Links
- https://monopedia-ui-git-media-prediqt.vercel.app/create-wiki?slug=sushiswap

# Screenshots

## Media Section Input
Frame 179

## Editor Plugin
Frame 178

### CODE REVIEW

File: src/components/Layout/Editor/Highlights/MediaModal/styles.ts [hunk 1]
```diff
@@ -0,0 +1,70 @@
+import { CSSObject } from '@chakra-ui/react'
```
Comment (anubra266) [inline src/components/Layout/Editor/Highlights/MediaModal/styles.ts:1]: This file is not used at all. In the tags file, you imported the styles used in the highlight modal styles. You can just delete this.

Comment (Emmanueldmlr) [reply src/components/Layout/Editor/Highlights/MediaModal/styles.ts]: Deleted. Thanks

**Comment by Emmanueldmlr** [comment]
@Royal-lobster, I already updated the frame to use the new type.

**Comment by Emmanueldmlr** [comment]
@kesar @Royal-lobster , kindly review and test out the media section and widget.

**Pull request merged**